### PR TITLE
Item updates + some others

### PR DESCRIPTION
**Pull request by DabirSA**

### CODE REVIEW

File: json/Name_Actor_MagName.txt [hunk 1]
```diff
@@ -817,62 +817,62 @@
...
 		"assign": "4163",
 		"jp_text": "ラブ・エンペ",
-		"tr_text": ""
+		"tr_text": "Love Emperappy"
 	},
 	{
 		"assign": "4166",
 		"jp_text": "テルテルラッピー",
-		"tr_text": ""
+		"tr_text": "Teru Teru Rappy"
```
Comment (alama) [inline json/Name_Actor_MagName.txt]: `テルテル・ラッピー` already have this name

Comment (DabirSA) [reply json/Name_Actor_MagName.txt]: It looks to me like this is a problem with the check, not with this name. This file is mag names for the mag feeding UI, there's no need to compare it against item names.

Comment (alama) [reply json/Name_Actor_MagName.txt]: Then you can remove `Name_Actor_MagName.txt` from `_tools/dupcheck.py`
While you are at it, remove the `Explain_Actor_MagAuto.txt` check

Comment (alama) [reply json/Name_Actor_MagName.txt]: Just to note, the item patch IS have a issue with `"テルテルラッピー` and `テルテル・ラッピー` both having "Teru Teru Rappy"

**Pull request merged**